manifests/imagestream.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  name: peribolos-service-controller
+spec:
+  lookupPolicy:
+    local: true
```

Comment:
I think this imagestream is missing some `spec` content. We need it to define a `latest` and point to quay, don't we? cc @harshad16 can you please help us understand what should be the Imagestream content for this to work?

Reply:
I think @KPostOffice  is waiting on the prod/dev/test overlays PR to come in 
as it the test/imagestream.yaml would have the image tag pointing to quay.
and the path of the imagestream.yaml would be added to the `.aicoe-ci.yaml`